javascript/browser-apis/toolbox-i/geolocation.md

```diff
@@ -97,7 +97,7 @@ Fill in the missing bits of the following snippet such that, using the Geolocati
 navigator.???.???(
   function (pos) {
     console.log('new position', pos)
-})
+});
```

Comment:
This isn't necessary and since all of the code isn't using `;`, might as well remove it

Reply:
It looks like most of our insights do this, so for the sake of consistency I just added them to everything.